sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/StorageTest.java

```diff
@@ -553,6 +555,32 @@ public void readFailedCosmosDbCallFailed() throws JSONException {
                 CoreMatchers.containsString("Cosmos db exception."));
     }
 
+    @Test
+    public void readCosmosDbCallEncodeDocumentId() throws JSONException, UnsupportedEncodingException {
+        String documentID = "Test Document";
+        AppCenterFuture<Document<TestDocument>> doc = Storage.read(USER, documentID, TestDocument.class);
```

Comment:
It's not allowed to commit with Android Studio warnings. Normally we use Android Studio commit dialog to prevent that.